lib/mix/gear_static_analysis.ex

```diff
@@ -75,75 +91,100 @@ defmodule Mix.Tasks.Compile.GearStaticAnalysis do
     {nil, issues ++ acc} # don't walk into the module body further by returning `nil`
   end
 
-  defp check_ast_node(n, file, tool?) do
-    case n do
-      {:defimpl, meta, [{:__aliases__, _, protocol_atoms}, [for: {:__aliases__, _, mod_atoms}], [do: _block]]} ->
-        # We don't have to check `defimpl` without `for:`, as the enclosing module's name is enforced to be properly prefixed by gear name.
-        check_defimpl(Module.concat(protocol_atoms), Module.concat(mod_atoms), meta, file)
-      {:use, meta, [{:__aliases__, _, atoms} | kw]} ->
-        with_concatenated_module_atom(atoms, fn mod ->
-          check_use_within_module(mod, kw, meta, file, tool?)
-        end)
-      {{:., _, [{:__aliases__, _, atoms}, fun]}, meta, args} ->
-        with_concatenated_module_atom(atoms, fn mod ->
-          check_remote_call(mod, fun, args, meta, file, tool?)
-        end)
-      {{:., _, [erlang_mod, fun]}, meta, args} ->
-        check_remote_call(erlang_mod, fun, args, meta, file, tool?)
-      {:__aliases__, meta, atoms} ->
-        with_concatenated_module_atom(atoms, fn mod ->
-          check_module(mod, meta, file, tool?)
-        end)
-      {fun, meta, args} when is_atom(fun) and is_list(args) ->
-        check_local_call(fun, args, meta, file, tool?)
-      atom when is_atom(atom) ->
-        check_atom(atom, file)
-      _ ->
-        nil
-    end
+  defp check_ast_node({:defimpl, meta, [{:__aliases__, _, protocol_atoms},
+                                        [for: {:__aliases__, _, mod_atoms}],
+                                        [do: _block]]},
+                      file,
+                      _tool?) do
+    concated_protocol_atoms = Module.safe_concat(protocol_atoms)
+    concated_mod_atoms = Module.safe_concat(mod_atoms)
+    # We don't have to check `defimpl` without `for:`, as the enclosing module's name is enforced to be properly prefixed by gear name.
+    check_defimpl(concated_protocol_atoms, concated_mod_atoms, meta, file)
   end
 
+  defp check_ast_node({:use, meta, [{:__aliases__, _, atoms} | kw]}, file, _tool?) do
+    with_concatenated_module_atom(atoms, fn mod ->
+      check_use_within_module(mod, kw, meta, file)
+    end)
+  end
+
```

Comment:
Please remove blank line.
https://github.com/rrrene/elixir-style-guide#group-function-definitions

Reply:
Thank you for the reference!
I fixed it in https://github.com/access-company/antikythera/pull/51/commits/321f10e26d7dc5be21f956fe70833dc4ce32588e